backend/server.py

```
@@ -7119,47 +7119,50 @@ async def workflow_orchestrator_status(
+            "by_status": {
+                "pending": sum(1 for r in runs if r.get("status") == "pending"),
+                "running": sum(1 for r in runs if r.get("status") == "running"),
+                "awaiting_approval": sum(1 for r in runs if r.get("status") == "awaiting_approval"),
+                "queued": sum(1 for r in runs if r.get("status") == "queued"),
+                "done": sum(1 for r in runs if r.get("status") == "done"),
+                "failed": sum(1 for r in runs if r.get("status") == "failed"),
+            },
+            "queue": queue_status,
+            "supervisor": supervisor_state,
+        }
+    except Exception as _exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)